src/main/java/camp/nextstep/racingcar/CarRacing.java

```diff
@@ -0,0 +1,70 @@
+package camp.nextstep.racingcar;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+import java.util.PrimitiveIterator;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class CarRacing {
+  Random random = new Random();
+  final int START_POSITION = 1;
+
+  public void start(int carNumber, int rounds) {
+    List<Integer> racingState = new ArrayList<Integer>(Collections.nCopies(carNumber, START_POSITION));
```

Comment:
```suggestion
    List<Integer> racingState = new ArrayList<>(Collections.nCopies(carNumber, START_POSITION));
```
java 7부터 다이아몬드 연산자를 사용하여 생략할 수 있습니다ㅎㅎ